dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md

```diff
@@ -0,0 +1,211 @@
+# Jakarta Concurrency TCK Open Liberty Implementation
+
+This project is an implementation of the Jakarta Concurrency Technology Compatibility Kit (TCK) running on Open Liberty.
+Before running this project it is recommended to read the documentation located in the base [TCK project](https://github.com/eclipse-ee4j/concurrency-api/blob/master/tck/README.md)
+
+## Overview
+
+In order to aid in the development of the `Concurrency-3.0` feature, this project is configured specifically to allow
+the feature developers to run the TCK against a development image of Open Liberty.
+
+## Requirements
+
+- [JDK 11 or higher](https://adoptopenjdk.net/?variant=openjdk11)
+- [Maven 3.6.0 or higher](https://maven.apache.org/download.cgi)
+
+## Getting started
+
+### Building Jakarta Concurrency
+
+If you want to test against the latest API and TCK versions the first step will be to clone and build the Jakarta Concurrency project.
+
+```sh
+git clone git@github.com:eclipse-ee4j/concurrency-api.git
+cd concurrency-api
+mvn clean install
+```
+
+The API and TCK libraries will be tagged withe `3.0.0-SNAPSHOT` version this keep this in mind if you plan to follow the [Running the TCK for Verification](#Running-the-TCK-for-Verification) section. 
+
+### Getting Open Liberty
+
+You will first need to clone the Open Liberty project:
+
+```sh
+git clone git@github.com:OpenLiberty/open-liberty.git
+cd open-liberty/dev
+```
+
+## Choose a test strategy
```

Comment:
This section is important to be aware of almost from the beginning.  Can we use it as the "Overview"?  This will help it to make the most sense to the release approvers who are looking at this as documentation of how to run the official TCK against the Open Liberty beta.  That use case should be the most prominent focus, with the develop use case secondary.